quic/s2n-quic-core/src/recovery/bbr.rs

```diff
@@ -1083,7 +1124,59 @@ impl congestion_controller::Endpoint for Endpoint {
         &mut self,
         path_info: congestion_controller::PathInfo,
     ) -> Self::CongestionController {
-        BbrCongestionController::new(path_info.max_datagram_size)
+        BbrCongestionController::new(path_info.max_datagram_size, self.app_settings)
+    }
+}
+
+pub mod builder {
+    use super::{ApplicationSettings, Endpoint};
+
+    /// Build the congestion controller with application provided overrides
+    #[derive(Debug, Default)]
+    pub struct Builder {
+        cwnd: Option<u32>,
+        cwnd_gain: Option<u32>,
+        loss_threshold: Option<u32>,
+        up_pacing_gain: Option<u32>,
+    }
+
+    impl Builder {
+        /// Set the initial congestion window.
+        pub fn with_congestion_window(mut self, cwnd: u32) -> Self {
+            self.cwnd = Some(cwnd);
+            self
+        }
+
+        /// The dynamic gain factor used to scale the estimated BDP to produce a congestion window (cwnd)
```

Comment:
the comment for this one and the next two should indicate how its used as a numerator in a ratio over 100